components/SpacesCard.js

```diff
@@ -0,0 +1,54 @@
+import React from "react";
+import { Paper, Container, Box, Typography } from "@material-ui/core";
+import { useTheme } from "@material-ui/core/styles";
+import VideocamOffIcon from "@material-ui/icons/VideocamOff";
+import MicIcon from "@material-ui/icons/Mic";
+import ChatIcon from "@material-ui/icons/Chat";
+import PeopleIcon from "@material-ui/icons/People";
+import LibraryMusicIcon from "@material-ui/icons/LibraryMusic";
+import { AddBoxOutlined, AddBoxRounded } from "@material-ui/icons";
+import { Children } from "react";
+
+export default function SpacesCard(props) {
+  const theme = useTheme();
+
+  return (
+    <Paper
+      className={`flex flex-col`}
```

Comment:
why curly braces and back-tick?

Reply:
might be nice to add `rouded-small` as a class name. You may have to define a border though